app/controllers/catalog_controller.rb

```diff
@@ -1497,14 +1497,14 @@ def available_job_templates(manager_id)
 
   def fetch_all_templates(manager_id)
     all_templates = ExtManagementSystem.find_by(:id => manager_id).configuration_scripts.sort_by(&:name)
-    job_templates = all_templates.collect { |t| [t.name, t.id] if t.type == 'ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScript' }.compact
-    workflow_templates = all_templates.collect { |t| [t.name, t.id] if t.type == 'ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationWorkflow' }.compact
+    job_templates = all_templates.collect { |t| [t.name, t.id] if t.kind_of?(ManageIQ::Providers::Awx::AutomationManager::ConfigurationScript) }.compact
+    workflow_templates = all_templates.collect { |t| [t.name, t.id] if t.kind_of?(ManageIQ::Providers::Awx::AutomationManager::ConfigurationWorkflow) }.compact
```

Comment:
This is likely going to cause issues with AnsibleTower because tower ConfigurationScripts and ConfigurationScriptWorkflows won't match on this.  Adding a `kind_of?(Awx) || kind_of?(Tower)` isn't the right answer here either.

I think the only way to do this properly is to add a `ManageIQ::Providers::AutomationManager::ConfigurationWorkflow` class to core, change `ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationWorkflow` to inherit from that instead of `ManageIQ::Providers::AutomationManager::ConfigurationScript`, then changing these to look for the "base" types.

Reply:
@agrare sounds like a good idea, I'll make the change